### DISCUSSION TITLE
Allow GPU/CPU choice in manager instead of project webpage's venue

### DISCUSSION THREAD

**Discussion by hucker75** [BOINC/boinc#5023] [Ideas]
When running several machines on several projects, I often want some of them to use only the CPU, or only the GPU, or both, on different projects.  It would be easier if I could control this on the computers instead of having to change the venue on the project webpage.

**Comment by davidpanderson** [comment]
There's a config option <exclude_gpu> that lets you disable GPU for a particular project:
https://boinc.berkeley.edu/wiki/Client_configuration#Options
There's no <exclude_cpu> option, though that wouldn't be hard to add.

**Reply by hucker75** [reply]
> There's a config option <exclude_gpu> that lets you disable GPU for a particular project: https://boinc.berkeley.edu/wiki/Client_configuration#Options

I use that when I have differing GPUs on a host.  But it's just as much work to use that to disable all of them as to go to the project's website to turn off GPU work.  An option in the Boinc Manager to make the host use only the CPU, or only the GPUs, for a certain project, by just clicking something in a menu would be good.

> There's no <exclude_cpu> option, though that wouldn't be hard to add.

As I only use <exclude_gpu> when I have differing GPUs in a system, I can't think of when <exclude_cpu> would be useful, since a system has only 1 cpu, or 2 identical ones.
